compiler/src/dotty/tools/dotc/transform/init/Semantic.scala

```diff
@@ -792,10 +810,7 @@ object Semantic {
 
           case _ => default()
         }
-  end extension
-
-// ----- Promotion ----------------------------------------------------
-  extension (ref: Ref)
+    // ----- Promotion ----------------------------------------------------
```

Comment:
Nitpick: accidentally indented the section comment? BTW, now `accessLocal` and `isFullyFilled` share the same extension group. It would be better to keep the original extension group unchanged.